### PR TITLE
Creating the header and the first section #1

### PR DESCRIPTION
**Pull request by talaalsweiti**
- [x]  Created Header with navbar and logo.

- [x]  created the first section of the landing page.
attached screenshot for my work 
Screenshot 2020-10-10 214840

### CODE REVIEW

**Comment by saharAdem** [comment]
Good Job Tala, your page is not responsive. please do it.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<body>
+    <header>
+        <div class="logo-container">
+
+            <p>Evolo</p>
+
+        </div>
+        <nav>
+            <ul>
+                <li>Home</li>
```
Comment (saharAdem) [inline index.html]: Each list item must be a link. So, add an anchor for each one. 

`            <li><a href="#">Home</a></li>
`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    
+    .content span:nth-child(2) {
+    font-size: 40px;
+    color: #393939;
+    font-weight: bold;
+    }
+
+   .content .btn {
+    display: inline-block;
+    padding: 22px 24px;
```
Comment (saharAdem) [inline style.css:112]: You wrote the padding another time below. So remove this one

**Review by Rahaf-96 — APPROVED** [review]
Looks Great! :partying_face: but you need to work more on the responsive part.

**Review by waed-hamza — APPROVED** [review]

**Review by saharAdem — APPROVED** [review]

**Pull request merged**